elide-contrib/elide-dynamic-config-helpers/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/DynamicConfigHelpers.java

```diff
@@ -190,4 +156,40 @@ public static String readConfigFile(File configFile) {
         }
         return sb.toString();
     }
+
+    /**
+     * Read config from classpath.
+     * @param resourcePath : path to resource
+     * @return content of resource
+     * @throws IOException
+     */
+    public static String readResource(String resourcePath) throws IOException {
```

Comment:
I think we can remove this function and also readConfigFile (see later comment).